src/service/donateService.js

```diff
@@ -0,0 +1,12 @@
+const Donate = require('../model/Donate');
+
+const create = async (donate) => {
+    await Donate.create(donate)
+    .then(donate => {
+        return donate;
+    });
+};
```

Comment:
then 을 쓰는 이유와 그 결과괎을 return 하지 않는 이유가 있을까요?

```suggestion
const create = async (donate) => {
    const createDonate = await Donate.create(donate);
    return createDonnate;
};
```

Reply:
오우 세상에 회사에서 몰래 이것저것 해보다가 이상하게 짰네요 감사함다...